src/sentry/constants.py

```diff
@@ -168,7 +168,6 @@ def get_all_languages():
     'password',
     'secret',
     'passwd',
-    'authorization',
```

Comment:
Since we're removing this, do you mind adding this to the tests? If not, I can just take over the PR and amend. Should be trivial, I just want to make sure that we don't even regress on this behavior in the future if say, `auth` were for some reason removed.